eval/eval.go

```diff
@@ -1419,3 +1427,11 @@ func (s *State) stopOutputBuffering() []byte {
 	s.env.PrevOut = nil
 	return output
 }
+
+func isAssignment(tok token.Type) bool {
+	return tok == token.ASSIGN || tok == token.DEFINE || isCompound(tok)
+}
+
+func isCompound(tok token.Type) bool {
+	return (tok > token.DEFINE && tok < token.FUNC)
```

Comment:
ahem... as mentioned ... please use >= SUMASSIGN <= SUMDIV and not the unrelated token.FUNC
(which btw also includes a marker gap)